demo/settings_USA.yaml

```diff
@@ -42,6 +42,9 @@ search:
   company_block_list:
     - "Infox Consulting"
 
+  # The desired level of work-remoteness (i.e. IN_PERSON, FULLY_REMOTE, ANY)
```

Comment:
We should probably document all of the options for remoteness here or somewhere else for users who might not look at the code.

Reply:
:+1: I've added all the options to this docstring, they are also available by CLI `-h`